qrenderdoc/Windows/ShaderViewer.cpp

```diff
@@ -516,9 +516,6 @@ void ShaderViewer::debugShader(const ShaderBindpointMapping *bind, const ShaderR
 
         QString semIdx = s.needSemanticIndex ? QString::number(s.semanticIndex) : QString();
 
-        QString regIdx =
-            s.systemValue == ShaderBuiltin::Undefined ? QString::number(s.regIndex) : lit("-");
-
         ui->inputSig->addTopLevelItem(new RDTreeWidgetItem(
             {name, semIdx, QString::number(s.regIndex), TypeString(s), ToQStr(s.systemValue),
```

Comment:
This one is a legit copy-paste bug, it should have been used - the `QString::number(s.regIndex)` below should actually be `regIdx` - see the similar path below filling `outputSig`.

Reply:
Done.